translate/storage/rc.py

```diff
@@ -308,7 +308,7 @@ class rcfile(base.TranslationStore):
     """This class represents a .rc file, made up of rcunits."""
 
     UnitClass = rcunit
-    default_encoding = "cp1252"
+    default_encoding = "auto"
```

Comment:
Won't this cause regression for files that are actually cp1252? The autodetection between iso-8859-1 and cp1252 is not much reliable...

Reply:
I don't think so, because I don't think this value does much currently.

It seems to be passed as a kwarg to rcfile.init() when rc.py is invoked by test_rc2po.py (but not rc2po.py, which has it's own setting for this), but is just ignored as it's not passed into rcfile.parse(), so we're currently always relying on the autodetection there.

Reply:
Looking at this a bit more, I think I see what you mean: this does possibly change the encoding automatically selected if rcfile.__init__() is called without an encoding kwarg.

I adjusted things so 'cp1251' is the default_encoding to be tried if charset_normalizer can't determine one.

Reply:
Going again through the code, `default_encoding` should be real encoding:

https://github.com/translate/translate/blob/4d12d3737241e6de6d8cc180179b06af5f752bda/translate/storage/base.py#L771

Reply:
I hadn't noticed that usage, but I guess that explains something that had confused me about how some of the tests were behaving.

There seems to be a confusion of meanings of `default_encoding`.  Is it "the encoding used when one isn't specified" or "the fallback encoding to use when one isn't specified, we tried autodetection, and that failed"?